pyseir/deployment/webui_data_adaptor_v1.py

```diff
@@ -218,14 +218,34 @@ def map_fips(self, fips):
 
             # Truncate date range of output.
             output_dates = pd.to_datetime(output_model['date'])
-            output_model = output_model[ (output_dates > datetime(month=3, day=3, year=2020))
+            output_model = output_model[ (output_dates >= datetime(month=3, day=3, year=2020))
                                         & (output_dates < datetime.today() + timedelta(days=90))]
             output_model = output_model.fillna(0)
 
+            try:
+                rt_results = load_Rt_result(fips)
+                rt_results.index = rt_results['Rt_MAP_composite'].index.strftime('%m/%d/%y')
+
+                merged = output_model.merge(rt_results[['Rt_MAP_composite', 'Rt_ci95_composite']],
+                    right_index=True, left_on='date', how='left')
+
+                output_model[schema.RT_INDICATOR] = merged['Rt_MAP_composite']
+                # 90% confidence intervals run from 5% - 95%
+                output_model[schema.RT_INDICATOR_CI90] = merged['Rt_ci95_composite'] - merged['Rt_MAP_composite']
```

Comment:
How should we interpret the CI90:  with 90% probability the value is between rt_indicator - ci90 to rt_indicator + ci90?

Reply:
That is correct. Comment added.